app/app.js

```diff
@@ -96,9 +96,19 @@ app.get('/api/jobLogs/', (req, res) => {
 });
 
 //Channel testing 
-app.post('/api/addChannelConfigs/', function (req, res) {
+app.post('/api/addEmailConfigs/', function (req, res) {
```

Comment:
we do not need rest apis....please replace with grpc service

Reply:
rest API is needed for unit test case